### DISCUSSION TITLE
List of All published Modules?

### DISCUSSION THREAD

**Discussion by BicycleMark** [oqtane/oqtane.framework#1391] [General]
Hello Oqtane users/developers:
   Is there a centralized list of all available Modules for Oqtane?

Mark

**Comment by leigh-pointer** [comment]
Nuget is the centralized place. Also if you go to "module management" in the Oqtane framework you can see a list of modules available to install. You need to be logged in as host.

**Reply by BicycleMark** [reply]
thanks Leigh
